README.md

```diff
@@ -18,53 +18,24 @@ Beginning in version 0.3.0, projects which use sourcemaps to debug code transpil
 ---
 ## Usage
 
-### Lua Stand-Alone Interpreter
-To debug a Lua program using a stand-alone interpreter, set `lua-local.interpreter` in your user or workspace settings:
-
-!["lua-local.interpreter": "lua5.1"](resources/settings.png '"lua-local.interpreter": "lua5.1"')
+### Tarantool Lua Stand-Alone Interpreter
+To debug a Lua program using a stand-alone interpreter, set `lua-tarantool.interpreter` in your user or workspace settings.
```

Comment:
1. I assume we now should refer to `lua-tarantool.tarantool` parameter here;
2. I'd rephrase it slightly (t omake it more Tarantool specific), e.g:

"To debug a Lua program using particular Tarantool executable, set `lua-tarantool.tarantool` in your user or workspace settings."